parm/makefile

```diff
@@ -45,6 +46,8 @@ GEFSANLCTRLFILENAME     = postcntrl_gefs_anl.xml
 GEFSCTRLF00FILENAME     = postcntrl_gefs_f00.xml
 FV3LAMAVAILXMLFILENAME  = post_avblflds.xml
 FV3LAMCTRLFILENAME      = fv3lam.xml
+FV3LAMCMAQAVAILXMLFILENAME  = fv3lam_cmaq_post_avblflds.xml
```

Comment:
@hsinmulin-NOAA The avblflds.xml was unfiled. All new variables should be added in post_avblflds.xml.

Reply:
no additional vars are added, just separated CMAQ use to others

Reply:
The change is based on the original existed "fv3lam_post_avblflds.xml" and the original "makefile" structure. new "fv3lam_cmaq_post_avblflds.xml" & "fv3lam_cmaq.xml" and the modified "makefile" is to avoid the variables used in RRFS-CMAQ as dummy variables in LAM

Reply:
Updated.
Using unified "post_avblflds.xml"  per removal of "fv3lam_cmaq_post_avblflds.xml".